views/layout/default.blade.php

```diff
@@ -734,6 +736,8 @@ class="col content-text">
 	@if(!empty($__t('bootstrap-select_locale') && $__t('bootstrap-select_locale') != 'x'))<script src="{{ $U('/node_modules', true) }}/bootstrap-select/dist/js/i18n/defaults-{{ $__t('bootstrap-select_locale') }}.js?v={{ $version }}"></script>@endif
 	<script src="{{ $U('/node_modules/jquery-lazy/jquery.lazy.min.js?v=', true) }}{{ $version }}"></script>
 	<script src="{{ $U('/node_modules/nosleep.js/dist/NoSleep.min.js?v=', true) }}{{ $version }}"></script>
+	<script src="{{ $U('/node_modules/daterangepicker/daterangepicker.js?v=', true) }}{{ $version }}"></script>
+	<script src="{{ $U('/node_modules/canvasjs/dist/jquery.canvasjs.min.js?v=', true) }}{{ $version }}"></script>
```

Comment:
When a library is only needed on a single page, it's not needed to include it globally (like [here](https://github.com/grocy/grocy/blob/91700e7dae4b35c32c2483ce0837a714df87a398/views/calendar.blade.php#L7-L16)).

Reply:
Ah, yes. No need to declare it globally. I fixed this.